opal/corelib/main.rb

```diff
@@ -5,3 +5,8 @@ def self.to_s
 def self.include(mod)
   Object.include mod
 end
+
+# Compiler overrides this method
+def using(mod)
```

Comment:
I changed this to be `self.using` so it's defined only on main and not on Object